src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+* {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  background-color: #d4e2e6;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+  position: relative;
+
+  &-minute,
+  &-second {
+    position: absolute;
+    transform-origin: bottom;
+    bottom: 50%;
+  }
+
+  &-minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    animation: move 600s steps(60) infinite;
+  }
+
+  &-second {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    animation: move 10s linear infinite;
+  }
+
+  &-center {
+    position: absolute;
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 50%;
+    background-color: #f6a603;
+  }
+
+  &--speed-up {
```

Comment:
should be vise versa - add --speed-u modifier to the `&-second` and minute selectors